### ISSUE TITLE
Internal laptop screen issue after kernel upgrade 5.8 -> 5.11

### ISSUE THREAD

**Issue by CristianKerr** [pop-os/linux#67]
I have had this issue for about two months and I was already quite desperate:
My internal laptop screen stopped working when using Pop!_OS. It started on 20.10 and continued after updating to 21.04. Finally someone pointed out, that it was about the time, when you updated your linux kernel. 
So with version 5.11.0-7620-generic I have the issue (I also tried previous 5.1.0-something-generic, dont remember the number, but didn't worked either). Then I tried to boot with old kernel from 20.04 (5.8.0-7642-generic) and the issue is gone. 

So what happenes:
- screen is black most of the time - but back light works - and I can tune it up or down with fn keys
- only few times (very unfrequently) it glitched with scrambled pixels - vertical colorful lines
- latest ubuntu 21.04 doesn't suffer from the issue. Their kernel version is 5.11.0-16-generic - but I didn't manage to install this kernel to my pop.
- external monitor works fine- tried more than one
- the screen stops working when the encryption screen shows and on (password for cryptsetup)
- Laptop HP spectre x360 13 model 2017
- Intel integrated HD graphics 620 (KBL GT2)
- Mesa - core profile 4.6 Mesa 21.0.1 - I tried install latest version from custom repo and that didn't help

**Comment by CristianKerr** [comment]
Please let me know if I can provide some logs or anything, I am willing to help to catch this bug, I am just not experienced enough to know what else I need to post here.

**Comment by CristianKerr** [comment]
I just tested Ubuntu 21.10 and it doesn't have the issue either.

**Comment by jackpot51** [comment]
We have kernel 5.13 on 20.04 and 21.04 now, does the issue still happen with that?

**Comment by CristianKerr** [comment]
Yes, I have the latest kernel (and all updates) and the issue still persists. Last kernel on which it worked (and that I keep as an alternative that I often use) is 5.8.0-7642

**Comment by jackpot51** [comment]
What kernel version did you try on Ubuntu 21.10?

**Comment by CristianKerr** [comment]
Uh, I didn't check, I downloaded latest Ubuntu and ran it from the flash. I will check it tomorrow what version it is.

**Comment by CristianKerr** [comment]
So it's 5.13.0-19-generic

**Comment by CristianKerr** [comment]
Okay, now it started working after I got some updates. I checked with the previous kernel and it doesn't work with it so in latest version the issue is probably resolved. Thank you if its your doing!!!  I was fighting with this issue for almost half of the year almost ready to give up...
